src/sqlfluff/rules/layout/LT13.py

```diff
@@ -63,7 +63,9 @@ class Rule_L050(BaseRule):
             foo
     """
 
-    groups = ("all",)
+    name = "layout.start-of-file"
```

Comment:
shouldn't this be snake case?

Reply:
also, possibly a good test to add, as an easy mistake to make when writing a new rule

Reply:
oh - good catch. Yes it should 👍

Reply:
I like the idea of validating rule names too - I'll do that as a new PR though.